spec/forms/candidate_interface/volunteering_role_form_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe CandidateInterface::VolunteeringRoleForm, type: :model do
+  FeatureFlag.deactivate('restructured_work_history')
```

Comment:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/4213/files#r592392701